lnrpc/routerrpc/router_backend.go

```diff
@@ -104,6 +104,10 @@ type RouterBackend struct {
 	// TODO(yy): remove this config after the new status code is fully
 	// deployed to the network(v0.20.0).
 	UseStatusInitiated bool
+
+	// SetExperimentalEndorsement returns a boolean indicating whether
+	// the experimental endorsement bit should be set.
+	SetExperimentalEndorsement func() bool
```

Comment:
Maybe rename to `ShouldSetExperimentalEndorsement`? "Set" as a prefix confers an understanding of it mutating state which afaict this doesn't do.